recipes/nettle/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class NettleTLS(ConanFile):
+    name = "nettle"
+    description = "The Nettle and Hogweed low-level cryptographic libraries"
+    homepage = "https://www.lysator.liu.se/~nisse/nettle"
+    topics = ("conan", "nettle", "crypto", "low-level-cryptographic", "cryptographic")
+    license = ("GPL-2.0", "GPL-3.0")
```

Comment:
```suggestion
    license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
```

https://www.lysator.liu.se/~nisse/nettle/nettle.html#Copyright